frontend/src/app/services/covid.service.ts

```diff
@@ -38,4 +38,8 @@ export class CovidService {
       });
   }
 
+  public getProvinceSituationPerDate(): Promise<GeneralSituationDaily[]> {
+    return this.http.get(this.apiUrl + '/daily/provincesituation/10')
```

Comment:
The path is /daily/generalSituation with query prameter area=[1,2,3].
For now, you can ask for area with id 10. You can do it like that:
this.http.get<any>(this.apiUrl + '/daily/generalSituation', {params: {'area': ['10']})